### DISCUSSION TITLE
How to get Fields errors from useFieldArrays in remix-validation-form ?

### DISCUSSION THREAD

**Discussion by YassineOujaa77** [remix-run/remix#6541] [Q&A] [answered]
I'm trying to build a form using rremix-validation-form and I'm using zod for creating the schema validation 
this is my schema : 
```
z.object({
    subQuestions: z.array(
      z.object({
        id: z.string(),
        question: z.string().min(1),
        answers: z.array(
          z.object({
            id: z.string(),
            answer: z.string().min(1),
          })
        ),
      })
    ),
  })
  ```
  
  but I cannot get the answers errors in my form ?
  I searched in remix-validation-form and in useFieldArray there is no way I can get these fields' errors ?
  DO you have any idea how can I do that ?

**Comment by sergiodxa** [comment, marked as answer]
I suggest you to ask in [`remix-validation-form` issues](https://github.com/airjp73/remix-validated-form/issues) instead of here since that's specific of the library.

**Reply by YassineOujaa77** [reply]
Thank you
